### ISSUE TITLE
uuid datatype support

### ISSUE THREAD

**Issue by Follpvosten** [SeaQL/sea-query#6]
Both Postgres and MySQL have a first-class `uuid` type by now. MariaDB doesn't, so that's yet another issue of that nature.

Once #5 is merged, I can just create a `UuidType` unit struct and implement Iden on it to work around this, but it would be nice to have API support for that on ColumnDef as well, at least at some point in the future.

**Comment by tyt2y3** [comment]
Thank you for your input. I think on the schema side there is still the default value expression to consider.
Yes on MySQL side explicit conversion has to be made.
I think we dont need to add a new variant for Value, because in the end, UUID in string representation is sent across the wire.
So may be an `eq_uuid` function is a short hard for `UUID_TO_BIN(value)` and then on Postgres the `UUID_TO_BIN` function actually does nothing as conversion is implicit.

**Comment by tyt2y3** [comment]
@Follpvosten can you share your use case (schema, select, insert, update) perhaps in raw SQL and I'd think about how to implement it in SeaQuery?

**Comment by tyt2y3** [comment]
UUID support in `Value` has been added in [0.10.0](https://github.com/SeaQL/sea-query/releases/tag/0.10.0).
Feel free to reopen this issue if you have amendments!
